src/components/Icons/TwoLetterSymbolIcon.vue

```diff
@@ -0,0 +1,43 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2021 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <SVGIcon
+    class="reset-circle-icon"
+    width="16px" height="16px" viewBox="0 0 16 16"
```

Comment:
Why is the view box for this icon 16 while others are 14?

Reply:
It was just how I got the icon from Design. I did not touch this, because I would have to recalculate the positioning of the SVG items, which I cannot do 🤣